src/Stock.js

```diff
@@ -94,6 +97,9 @@ class Stock extends React.Component {
                     /> :
                     <h2>Invalid Ticker</h2>
                 }
+                 <div className = "stockInfo">
+                    <p>{this.state.stockTicker} Volume: {this.state.stockVolumeValues.at(0)}</p>
```

Comment:
Could you maybe make this text context dependent. So it can specify whether it is the Daily Volume or the volume from the last X minutes. I think it'll improve the readability by a lot. You should be able to track the state with a variable (whether its intraday or daily) and then use the ? notation for if statements to achieve this effect.

Reply:
Done! Volume text now specifies current time selection.